jenkins-plugin/src/main/resources/org/jenkinsci/plugins/pipeline/maven/GlobalPipelineMavenConfig/config.jelly

```diff
@@ -30,6 +30,9 @@ THE SOFTWARE.
         <f:entry title="${%DAO}">
             <pre><code>${instance.dao.toPrettyString()}</code></pre>
         </f:entry>
+        <f:entry title="${%No Data Storage}">
+            <f:checkbox title="${%Do not store any data}" field="noDataStorage"/>
```

Comment:
worth an inline help that says what features will not work (e.g. SNAPSHOT triggering)

Reply:
yup done